cmake/modules/InstallPipDep.cmake

```diff
@@ -0,0 +1,11 @@
+function(install_pip_requirements requirements_file outfile)
+  find_program(PYTHON "python")
+
+  if(PYTHON)
+    set(PIP_ARGS -m pip install -r ${requirements_file} --target ${outfile})
```

Comment:
OK, so we always go with `--target`. Is the user's responsibility to add this directory to `PYTHONPATH`, right?

Reply:
Yup, with this PR we want to add predefined dependency folders inside ceph. The python path would look like this:
```
./src/pybind/mgr
./src/pybind/deps
/usr/local/lib64/python3.10/site-packages
/usr/local/lib/python3.10/site-packages
/usr/lib64/python3.10/site-packages
/usr/lib/python3.10/site-packages
./src/pybind
./build/lib/cython_modules/lib.3
./src/python-common

/usr/lib64/python310.zip
/usr/lib64/python3.10
/usr/lib64/python3.10/lib-dynload
```

Where the deps folder of mgr would be the first place too look at for known dependencies.